queries.sql

```diff
@@ -8,3 +8,32 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true; 
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg between 10.3 and 17.4;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is null;
+COMMIT;
```

Comment:
* Kindly complete this transaction as the requirement states that you `Verify that change was made` and `persists after commit`. [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Commit%20the%20transaction,persists%20after%20commit.)